ngrinder-core/src/main/java/net/grinder/util/NetworkUtils.java

```diff
@@ -226,7 +223,13 @@ public static List<Integer> getAvailablePorts(String ip, int size, int from, int
 		if (StringUtils.isNotBlank(ip)) {
 			try {
 				inetAddress = InetAddress.getByName(ip);
-			} catch (Exception e) {
+				if (!inetAddress.isReachable(MAX_REACHABLE_TIMEOUT)) {
+					processException(
+						"Can not check available ports because given local IP address '" + ip + "' is unreachable. " +
```

Comment:
IP address. <-- . is missing.

Reply:
I was intended to make a message like `IP address '${ip}' is unreachable.`. 
In this case, exception message is printed like this.
```
Caused by: org.ngrinder.common.exception.NGrinderRuntimeException: Can not check available ports because given local IP address '123.234.123.234' is unreachable. Please check the `/etc/hosts` file or manually specify the local IP address in `${NGRINDER_HOME}/system.conf`.
	at org.ngrinder.common.util.ExceptionUtils.processException(ExceptionUtils.java:38)
	at net.grinder.util.NetworkUtils.getAvailablePorts(NetworkUtils.java:227)
	at org.ngrinder.perftest.service.ConsoleManager.init(ConsoleManager.java:76)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        ...
```